tools/create_user.py

```diff
@@ -0,0 +1,19 @@
+import argparse
+import db.users
+from werkzeug.security import generate_password_hash
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument("--username", required=True, nargs="?", type=str)
+    parser.add_argument("--password", required=True, nargs="?", type=str)
+    parser.add_argument("--role", required=True, nargs="?", type=str,
+                        choices=['admin', 'manager'])
```

Comment:
hm, I don't see these roles being used, what were your thoughts here on the distinction?

Reply:
Yeah I plan to introduce an admin user which can add another users, instead of doing it through the script like it is done now. Manager role would only be able to approve plugins, where admin would do that + adding users.